internal/utils/test/assert/assert.go

```diff
@@ -29,47 +29,52 @@ func RegisterOpts(t reflect.Type, opts ...cmp.Option) {
 
 // Equal compares x and y for equality
 // and fails with the test if not
-func Equal(t testing.TB, x, y interface{}) {
+func Equal(t testing.TB, expected, actual interface{}) {
 	t.Helper()
-	switch x.(type) {
+	switch expected.(type) {
 	case string:
-		Equalf(t, x, y, "expected %q to equal %q", y, x)
+		Equalf(t, expected, actual, "failed to assert equals ( actual, expected )\n\t%q\n\t%q", actual, expected)
```

Comment:
⚠️ the changes in this file are aesthetic... the diffs of expected vs actual are now lined up separately, making it a bit easier to visually compare right off the bat